linux/gnome/src/main/java/com/github/weisj/darkmode/platform/linux/gnome/GtkTheme.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2020 Jannis Weis
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ */
+package com.github.weisj.darkmode.platform.linux.gnome;
+
+public class GtkTheme {
+    private String name;
```

Comment:
Does name really need to be mutable? This whole class could become
````kotlin
data class GtkTheme(val name : String) : Comparable<GtkTheme> {
    override fun compareTo(other: GtkTheme): Int = name.compareTo(other.name)
}
````
in kotlin. I threw in an implementation for comparable just to make the ordering of entries in the resulting choices list more reproducible.

Reply:
That's perfect. I've only used Kotlin briefly so I forgot that a data class would be suitable for this.